src/pepr/policies/security.ts

```diff
@@ -56,9 +54,9 @@ When(a.Pod)
 When(a.Pod)
   .IsCreatedOrUpdated()
   .Mutate(request => {
-    if (exemptPrivileged(request)) {
+    markExemption(Policy.RequireNonRootUser)(request);
+    if (request.HasAnnotation(`uds-core.pepr.dev/uds-core-policies.${Policy.RequireNonRootUser}`))
       return;
```

Comment:
This bothers me for whatever reason - but `npx pepr format` doesn't change it. Any reason for dropping the explicit `{}` around return here (and the other spot or two where this same logic is used)?

Reply:
ahh yeah, when it sits on one line i think it looks nice, but since it got pushed down looks weird and easy to misread. I'll add the `{}`